src/groupByKey.ts

```diff
@@ -2,6 +2,23 @@ type GroupsMap<T> = {
   [key: string]: T[];
 };
 
-export function groupByKey(items, key) {
-  // write code here;
+export function groupByKey(
+  items: Array<object>,
```

Comment:
```suggestion
  items: Array<T>,
```

better use `T`, `object` can be a little bit confusing